lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -35,6 +35,7 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
         self.assertIsNotNone(response.data['courseware_summary'])
         for chapter in response.data['courseware_summary']:
             self.assertIsNotNone(chapter)
+        self.assertIsNotNone(response.data['studio_url'])
```

Comment:
I think you could maybe add in an actual check for the value here vs just asserting not None.

Side note, please also add in a check about the certificate data.

Reply:
the certificate data is null a lot of the time, i'm not sure what the best way to test i t is

Reply:
Hmm. I guess we might be able to assume the function is being properly tested elsewhere. Just make sure you can test it out with mock data locally at least.